packages/kmath/src/geometry.ts

```diff
@@ -21,12 +23,9 @@ export type SineCoefficient = [
     number, // verticalOffset
 ];
 
-// a, b, c
-export type QuadraticCoefficient = [number, number, number];
-
 // Given a number, return whether it is positive (1), negative (-1), or zero (0)
 export function sign(val: number): 0 | 1 | -1 {
-    if (eq(val, 0)) {
+    if (approximateEqual(val, 0)) {
```

Comment:
I found out (unfortunately pretty late) that `eq` and `deepEq` aren't just standard equality checks, they're APPROXIMATE equality checks 😬 I thought it was worth being clear about that distinction.